app/views/assignments/_read.html.erb

```diff
@@ -104,15 +104,4 @@
       <% end -%>
     </div>
   <% end %>
-
-  <h3><%= t('assignment.required_files_read') %></h3>
```

Comment:
I'm not sure why this was changed. Is this view currently not used anywhere else?